ReleaseNotes.md

```diff
@@ -7,6 +7,33 @@ samples distributed with the code.  The samples demonstrate the usage of compres
 public models and datasets for three different use cases: Image Classification, Object Detection,
 and Semantic Segmentation.
 
+## New in Release 2.0:
+- Added TensorFlow 2.4.2 support - NNCF can now be used to apply the compression algorithms to models originally trained in TensorFlow. 
+Limitations apply, and feature parity of the TF backend with the PyTorch will be improved in the upcoming releases.
+Current NNCF TF features include:
+    - Quantization (with HW-specific targeting aligned with PyTorch, custom layers not supported)
```

Comment:
"INT8 Quantization" is better to avoid any confusion with INT4/2.

Reply:
Done